app/views/admin/users/index.html.erb

```diff
@@ -46,16 +36,7 @@
               <% if user.registration_facility.present? %>
                 Registration facility:
                 <% facility = user.registration_facility %>
-
-                <% if current_admin.permissions_v2_enabled? %>
-                    <%= link_to facility.name, [:admin, facility.facility_group, facility], class: "text-dark" %>
-                <% else %>
-                  <% if policy([:manage, :facility, facility]).show? %>
-                    <%= link_to facility.name, [:admin, facility.facility_group, facility], class: "text-dark" %>
-                  <% else %>
-                    <%= user.registration_facility.name %>
-                  <% end %>
-                <% end %>
+                <%= link_to facility.name, [:admin, facility.facility_group, facility], class: "text-dark" %>
```

Comment:
Same question as before - does our new scoping ensure you'll have access to this facility?

Reply:
Yes